src/components/map/Map.scss

```
@@ -0,0 +1,24 @@
+// *{
+//     border: 2px solid red;
+// }
+
+main{
```

Prefer space before brace
```suggestion
main {
```